File: test/fixtures/java-unirest/get_with_all_numerical_data.java
import kong.unirest.HttpResponse;
import kong.unirest.Unirest;

class Main {
    public static void main(String[] args) {
            HttpResponse<String> httpResponse
               = Unirest.post("http://localhost:28139/CurlToNode")
            .header("Content-Type", "application/json")
            .header("Accept", "application/json")
            .body("18233982904")
            .asString();
    }
}

File: test/fixtures/node-superagent/get_with_all_numerical_data.js
import request from 'superagent';

request
  .post('http://localhost:28139/CurlToNode')
  .set('Accept', 'application/json')
  .type('json')
  .send('18233982904')
  .then(res => {
     console.log(res.body);
  });
